src/scripts/jsonutils/jsonutils.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strconv"
 
 	"github.com/a8m/envsubst"
+	colorjson "github.com/TylerBrock/colorjson"
```

Comment:
I'm curious as to why this needs to be aliased as `colorjson`, looks like the module is already named that

Reply:
Thats true, I was cycling through a different libraries for testing purposes. This is still in draft. For what ever reason, nothing I tried so far is actually coloring the output